packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js

```diff
@@ -58,6 +64,14 @@ export default class QuanticSearchBox extends LightningElement {
    * @defaultValue 5
    */
   @api numberOfSuggestions = 5;
+  /**
+   * Whether to render the search box using a [textarea](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea) element.
+   * The resulting component will expand to support multi-line queries.
+   * @api
+   * @type {boolean}
+   * @defaultValue false
+   */
+  @api textarea = false;
```

Comment:
should we name this property to something more in-line with the feature? ie: `expandableSearchbox` or `expandableSearchboxEnabled` ?

Reply:
It's in line with the same property name from Atomic.